buildSrc/build.gradle.kts

```diff
@@ -23,5 +23,5 @@ repositories {
 }
 
 dependencies {
-  implementation("biz.aQute.bnd:biz.aQute.bnd.gradle:6.1.0")
+  implementation("biz.aQute.bnd:biz.aQute.bnd.gradle:6.2.0")
```

Comment:
Missed this, we should just update it to libs.gradlePlugin.bnd

Reply:
Yep, actually we can use catalog here, but the code navigation doesn't work, I don't konw why.

Reply:
OK, I might accept the update and leave for now then.  Thanks.